README.md

```diff
@@ -98,28 +100,28 @@ let number = CardNumber(rawValue: "5555 5555 5555 4444")
 //Now create a card object. You can optionally validate a card.
```

Comment:
`Initialize` here as well?

Reply:
I'd leave it – create an object, initialise an operation